.changeset/beige-moles-glow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/circuit-ui': minor
+---
+
+Fixes missing aria-busy on Button Component
```

Comment:
Not much better wording here but basically we're making sure that aria attributes are only rendered when the button is a loading button. I'd suggest:

```suggestion
Fixed the conditional rendering of aria attributes in loading buttons.
```